### DISCUSSION TITLE
Getting empty result when using fractional "size" variable.

### DISCUSSION THREAD

**Discussion by agalegov** [kernc/backtesting.py#724] [General]
Hey!

So i am tryin simple momentum strategy on BTC/USD data. When i define size as a wholenumber, backtest run just fine. But when i try to define it as a fraction of equity, i get empty results like this:

![image](https://user-images.githubusercontent.com/111352212/186436252-d6cc8c27-ceea-4d0e-99bd-f61626cc026d.png)

This is how i run the code:

`class MomTrade(Strategy):
    # Define timeperiod for momentum
    n = 15

    
    def init(self):
        # Precompute Momentum indicator
        self.MOM = self.I(MOM, self.data.Close, self.n)
        self.MOM1 = self.I(MOM, self.MOM, 1)
    
    def next(self):
        # If mom crosses 0, close any existing short trades, and buy the asset
        if crossover(self.MOM, 0): # and self.MOM1 > 0:
            self.position.close()
            self.buy(stop=self.data.High+0.01, size = 0.1)

        # Else, if momentum crosses 0 fron the top, reverse the position
        elif crossover(0, self.MOM): #and self.MOM1 < 0:
            self.position.close()
            self.sell(stop=self.data.Low+0.01, size = 0.1)

bt = Backtest(df, MomTrade, cash=100000, commission=0)
stats = bt.run()
display(stats)`

What am i doing wrong?

**Comment by kernc** [comment]
> ![image](https://user-images.githubusercontent.com/111352212/186436252-d6cc8c27-ceea-4d0e-99bd-f61626cc026d.png)

`Equity Final [$]` indicates you were, in fact, running it all with `cash=10_000`, which likely isn't sufficient for a unit of bitcoin in this time period.

See also: https://github.com/kernc/backtesting.py/issues/134
